backend/DAO/studentDAO.js

```diff
@@ -12,7 +12,7 @@ module.exports = {
         
         // This one is very similar to SQL
         let result = await database('student')
-            .select('id', 'name','surname')
+            .select('id', 'name','surname', 'imgurl')
```

Comment:
Is this still needed considering the fact that we won't allow the student to have an image?

Reply:
If it's not used in the frontend and doesn't break anything, I remove it

Reply:
yes, you can remove it from there and also from the seeds if you have time, it doesn't break anything in any story

Reply:
Okay I did this